arkouda/io.py

```diff
@@ -273,6 +277,46 @@ def get_datasets(filenames: Union[str, List[str]], allow_errors: bool = False) -
     return datasets
 
 
+def ls_csv(filename: str, col_delim: str = ",") -> List[str]:
+    if not (filename and filename.strip()):
+        raise ValueError("filename cannot be an empty string")
```

Comment:
It definitely should be. However, I believe there are cases where we can get `None` for filename, which this prevents. The csv case does not currently have these cases, but I think it would be good to leave, since it is modeled after our other `ls` function. 

I did confirm the `None` is able to be passed for `filename` and returns an error the `.strip()` is not valid for `NoneType`.

Reply:
ah okay! that makes sense!